### ISSUE TITLE
HTML in Checkbox Dialogue

### ISSUE THREAD

**Issue by MrVauxs** [trioderegion/warpgate#75]
Inserting HTML into the checkbox dialogue `label` property causes the formatting to break, due to it not being sanitized / not having an option to NOT have it added to the `name` of the checkbox itself.

```
{
            type: 'checkbox',
            label: `Turn your original token <u style="cursor: help;" title="Some token styles, particularly top-down ones, may look very bad whilst on top of your original token. Enable this to avoid that predicament.">invisible</u>?`
},
```

![image](https://user-images.githubusercontent.com/32039708/187524646-6ad02fb3-15c2-489b-af2f-0ae28a185307.png)
![image](https://user-images.githubusercontent.com/32039708/187524659-4f079c1e-dd94-4e2d-9677-450caa276f32.png)

**Comment by trioderegion** [comment]
Have you tried this on the v10 beta (1.14.!0)? It is backwards compatible with v9 and I have added `value` fields to both checkbox and radio inputs for exactly this reason -- use `label` with complex HTML and then provide `value` for the serializable result.

https://github.com/trioderegion/warpgate/tree/preview/v10#dialog -- see the `checkbox` row.

**Comment by MrVauxs** [comment]
Doesn't change anything.

**Comment by trioderegion** [comment]
Ah, indeed. This should be corrected in the new `1.14.0` full release. Thanks for the report.
